File: chatwoot/python/src/DeleteAgentFromAccountExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        api.AgentsApi(api_client).delete_agent_from_account(
            account_id=None,
            id=None,
        )
    except ApiException as e:
        print("Exception when calling AgentsApi#delete_agent_from_account: %s\n" % e)

File: chatwoot/ruby/src/DeleteAgentFromAccountExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["userApiKey"] = "USER_API_KEY"
end

begin
    ChatwootClient::AgentsApi.new.delete_agent_from_account(
        nil, # account_id
        nil, # id
    )
rescue ChatwootClient::ApiError => e
    puts "Exception when calling AgentsApi#delete_agent_from_account: #{e}"
end
